ansible/server-state-playbooks/www/tests/check_redirects.py

```diff
@@ -29,11 +29,18 @@
 
     ('/site/products', '/products'),
     ('/site/products/bio-formats', '/bio-formats'),
+    ('/site/products/bio-formats/downloads', '/bio-formats/downloads/'),
     ('/site/products/omero', '/omero'),
+    ('/site/products/omero/downloads', '/omero/downloads/'),
     ('/site/products/omero/feature-list', '/omero/new'),
+    ('/site/products/omero/secvuln', '/security/'),
+    ('/site/products/omero/secvuln/2014-SV3-csrf', '/security/2014-SV3-csrf/'),
 
     ('/site/support', '/docs'),
     ('/site/news', '/announcements'),
+
+    ('/info/vulnerabilities', '/security/'),
+    ('/info/vulnerabilities/2014-SV3-csrf', '/security/2014-SV3-csrf/'),
```

Comment:
Why is 2014 explicitly listed?

Reply:
This is related to the fact I defined 2 redirect rules (with or without security identifier). So these tests are testing each case i.e. `/security` with/out slash and `/security/YYYY-SVx-yyy` with/out slashes.